packages/server/postgres/queries/getTeamsByOrgIds.ts

```diff
@@ -0,0 +1,22 @@
+import {
+  getTeamsByOrgIdQuery,
+  IGetTeamsByOrgIdQueryParams,
+  IGetTeamsByOrgIdQueryResult
+} from './generated/getTeamsByOrgIdQuery'
+import getPg from '../getPg'
+
+const getTeamsByOrgIds = async (
+  orgIds: string[],
+  options: Partial<Omit<IGetTeamsByOrgIdQueryParams, 'orgId'>> = {}
```

Comment:
i think you want to omit `orgIds` here?
might just be simpler to write: options: {isArchived?: boolean} = {}